spec/factories/error_details.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+FactoryBot.define do
+  factory :error_detail do
+    organization
+    association :owner, factory: %i[invoice].sample
+    association :integration, factory: :anrok_integration
```

Comment:
Not sure it matters but I'd keep it empty by default and add a trait or a dedicated `anrok_error_detail` factory I think.

Reply:
Or like this: 

```suggestion
    association :integration, factory: %i[netsuite_integration okta_integration xero_integration anrok_integration].sample
```

Reply:
👍